ovos_padatious/padaos.py

```
@@ -1,39 +1,79 @@
 import re
 from threading import Lock
+from typing import Dict, List, Optional, Tuple, Union
```

Ruff

`typing.Tuple` imported but unused

Remove unused import: `typing.Tuple`

(F401)